pkg/e2e/adhoctestimages/adhoctestimages.go

```diff
@@ -45,23 +56,122 @@ var _ = ginkgo.Describe("Ad Hoc Test Images", ginkgo.Ordered, ginkgo.ContinueOnF
 
 		exe, err = executor.New(logger, exeConfig)
 		Expect(err).NotTo(HaveOccurred())
-		logger.Info("executing test suites", "suites", testImages)
+
+		// Log test images with their slack channels
+		imageNames := make([]string, len(testImages))
+		for i, img := range testImages {
+			imageNames[i] = img.Image
+		}
+		logger.Info("executing test suites", "suites", imageNames)
 	})
 
 	ginkgo.DescribeTable("execution",
-		func(ctx context.Context, testImage string) {
+		func(ctx context.Context, testImageConfig config.AdHocTestImage) {
+			testImage := testImageConfig.Image
 			baseImageName := strings.Split(testImage[strings.LastIndex(testImage, "/")+1:], ":")[0]
 			exeConfig.OutputDir = filepath.Join(viper.GetString(config.ReportDir), viper.GetString(config.Phase), baseImageName)
 
-			logger.Info("running test suite", "suite", testImage, "timeout", exeConfig.Timeout)
+			logger.Info("running test suite", "suite", testImage, "slackChannel", testImageConfig.SlackChannel, "timeout", exeConfig.Timeout)
```

Comment:
This log tells user which test suite is running. Slack channel (or any additional properties about the image) won't be useful here. You can add a debug message below if you need.